webapps/api/backend/views/albabackends.py

```diff
@@ -112,12 +112,12 @@ def add_units(self, albabackend, asds):
     @required_roles(['read', 'manage'])
     @return_task()
     @load(AlbaBackend)
-    def get_config_metadata(self, albabackend):
+    def get_arakoon_config(self, albabackend):
```

Comment:
I'd suggest to revert this particular line. It will be less changes in the openvstorage/framework repo as well.